crates/node/src/txvalidation/event.rs

```diff
@@ -202,3 +273,275 @@ impl TxEvent<NewTx> {
             .await
     }
 }
+
+// Use to cache New tx and store waiting Tx that have missing parent.
+pub struct TXCache {
```

Comment:
Should this possibly be named `TxCache` so that it's aligned with other types (e.g. `TxEvent` or `WaitTx` etc.)?